pkg/apis/certificates/types.go

```diff
@@ -43,6 +43,10 @@ type CertificateSigningRequestSpec struct {
 	// Base64-encoded PKCS#10 CSR data
 	Request []byte `json:"request"`
 
+	// signingProfile specifies a requested signing profile for a CA to use
+	// when issuing the certificate.
+	// SigningProfile string `json:"signingProfile,omitempty"`
```

Comment:
I think it is more comparable to pod security policy, and we don't make pod specs reference a pod security policy. Pod specs try to set a security context, and PSP allows or rejects. I'd expect CSRs to request usages, and an approver approves or denies (maybe in concert with predefined signing profiles)